services/client-react/README.md

```diff
@@ -29,7 +29,7 @@ OR
 ./scripts/setup-local-db
 ```
 
-If bash doesn't recognize the **REACT_APP_TASKS_SERVICE_URL** environment variable just export it and run the last two commands again.
+If the bash doesn't recognize the **REACT_APP_TASKS_SERVICE_URL** environment variable, just export it and run the last two commands again.
```

Comment:
"the " is not needed here, please delete.

Reply:
Thank you for reviewing! Resolved the issue.

Reply:
Thank you for reviewing! Resolved the issue.